src/pages/UsedCardScreen.tsx

```diff
@@ -1,19 +1,33 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import Prose from '../components/Prose'
 import Main, { MainChild } from '../components/Main'
 import Screen from '../components/Screen'
+import { PartialUserSettings } from '../config/types'
 
-const UsedCardScreen = () => (
-  <Screen>
-    <Main>
-      <MainChild center>
-        <Prose textCenter>
-          <h1>Used Card</h1>
-          <p>Please return card to a poll worker.</p>
-        </Prose>
-      </MainChild>
-    </Main>
-  </Screen>
-)
+interface Props {
+  setUserSettings: (partial: PartialUserSettings) => void
+}
+
+const UsedCardScreen = ({ setUserSettings }: Props) => {
+  useEffect(() => {
+    setUserSettings({ textSize: 3 })
+    return () => {
+      setUserSettings({ textSize: 1 })
+    }
+  }, [setUserSettings])
```

Comment:
Should this whole thing be a custom `useX`?

Reply:
Yeah… that might be better. I'd like to put the effort into themes rather than further support this clever hack. 😝